src/SprykerEco/Zed/UnzerApi/Business/Api/Response/UnzerApiResponseConstants.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerApi\Business\Api\Response;
+
+interface UnzerApiResponseConstants
```

Comment:
Please get rid of non-public API constants files:
1. Please use {Transfer}::PROPERTY where possible. And protected constants directly in the mapper were not.